app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,8 @@
+<% provide(:title, "The Title") %>
```

Comment:
`"The Title"`
em tìm hiểu I18n và sử dụng I18n trong toàn bộ file html nhé
Em self check sửa lại toàn bộ nhé
https://guides.rubyonrails.org/i18n.html

Reply:
.